examples/player/simid_player.js

```diff
@@ -430,6 +436,7 @@ class SimidPlayer {
 
     const creativeData = {
       'adParameters' : document.getElementById('ad_params').value,
+      'duration' : document.getElementById('duration').value,
```

Comment:
I think you only want to add this for nonlinear ads?

Reply:
good catch